BSPConversionLib/Source/EntityConverter.cs

```diff
@@ -298,12 +386,29 @@ private void ConvertTriggerMultiple(Entity trigger)
 					case "target_init":
 						ConvertInitTrigger(trigger, target);
 						break;
+					case "func_door":
+						ConvertDoorTrigger(trigger, target);
+						break;
 				}
 			}
 
 			trigger["spawnflags"] = "1";
 		}
-		
+
+		private void ConvertDoorTrigger(Entity trigger, Entity target)
```

Comment:
This should be named `OpenDoorOnStartTouch`. `target` should be renamed to `door`.